timvt/main.py

```diff
@@ -56,7 +56,7 @@ async def shutdown_event():
 
 
 # Register endpoints.
+app.include_router(demo.router, tags=["demo"])
```

Comment:
Demo has to be first because of `/index.json` (or tilejson.json will be conflicting)